py/torch_tensorrt/dynamo/conversion/impl/argmax.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional
+
+import tensorrt as trt
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
+from torch_tensorrt.dynamo.conversion.converter_utils import (
+    cast_trt_tensor,
+    flatten_dims,
+    get_axes_for_reduce_op,
+)
+from torch_tensorrt.fx.converters.converter_utils import (
+    get_positive_dim,
```

Comment:
Consider switching to `from torch_tensorrt.dynamo.conversion.converter_utils import get_positive_dim`